docs/usage.md

```diff
@@ -1,153 +1,8 @@
 # nf-core/nanoseq: Usage
```

Comment:
it seems all usage.md are cleaned, is this correct?

Reply:
I think he got all the flags off from the usage.